packages/eas-cli/src/commands/workflow/validate.ts

```diff
@@ -0,0 +1,83 @@
+import fs from 'fs/promises';
+import yaml from 'js-yaml';
+import path from 'path';
+
+import EasCommand from '../../commandUtils/EasCommand';
+import Log from '../../log';
+import { ora } from '../../ora';
+
+export class WorkflowValidate extends EasCommand {
+  static override description = 'validate a workflow configuration yaml file';
```

Comment:
```suggestion
export class WorkflowValidate extends EasCommand {
  static override description = 'validate a workflow configuration yaml file';
  static override hidden = true;
```
Let's hide it in our autogenerated docs + help command for now, I guess it's not top secret, but if it's not ready it's better to not put it there.